core/backend.py

```diff
@@ -18,7 +18,7 @@
 
 
 def read_image(image_data):
-    image = Image.open(io.BytesIO(image_data))
+    image = Image.open(io.BytesIO(image_data)).convert("RGB").convert("RGB")
```

Comment:
Do you actually need two `.convert("RGB")`

Reply:
No, huh I don't know how that happened